src/frequenz/client/microgrid/_component_states.py

```diff
@@ -80,6 +80,9 @@ class EVChargerComponentState(Enum):
     INTERRUPTED = ev_charger.ComponentState.COMPONENT_STATE_INTERRUPTED
     """The component is interrupted."""
 
+    UNKNOWN = ev_charger.ComponentState.COMPONENT_STATE_UNKNOWN
+    """The state is provided by the device, but it is not one of the above states."""
```

Comment:
I think we should say component, not device.

```suggestion
    """A state is provided by the component, but it is not one of the above states."""